kalite/updates/download_track.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from django.conf import settings
+logging = settings.LOG
+
+from .settings import VIDEO_DOWNLOAD_QUEUE_FILE
+
+class VideoQueue:
```

Comment:
I had a similar idea, BTW, that @tk21 is now working on -- [see doc on assessment item download during setup](https://docs.google.com/document/d/1ALHs6Ry3vxbd5TS--HkL9fqEWcR7XNWzZACs7n_ALE0/edit#heading=h.low45iy0i5at). But now I'm also wondering how the plans for that are affected by using content DBs? Do we have a concept of downloading assessment items at runtime, e.g. during upgrade, or is that replaced by downloading a content db?